app/src/main/java/it/quip/android/fragment/InviteFriendsFragment.java

```diff
@@ -54,7 +54,7 @@ public void onItemSelected(int position, View view) {
             }
         });
 
-        ListView lvInvitedFriends = (ListView) v.findViewById(R.id.lvInvitedFriends);
```

Comment:
so are we using snakecase for xml item names, and camelCase for everything else?